wgengine/router/callback.go

```diff
@@ -24,9 +24,17 @@ type CallbackRouter struct {
 	// will return ErrGetBaseConfigNotSupported.
 	GetBaseConfigFunc func() (dns.OSConfig, error)
 
-	mu   sync.Mutex    // protects all the following
-	rcfg *Config       // last applied router config
-	dcfg *dns.OSConfig // last applied DNS config
+	mu sync.Mutex // protects all the following
+
+	// InitialMTU is the MTU the tun should be initialized with.
+	// Zero means don't change the MTU. Once set, this field is
+	// zeroed out. This allows the MTU to be changed after
+	// initialization by other means (such as ifconfig). This
+	// field should be set only on creation and zeroed after the
+	// first use while holding mu.
+	initialMTU uint32
```

Comment:
If this isn't a public field, I think that the macOS/iOS code can't set this, so it'll always be zero (below)?

Reply:
Oh right, the user is in corp... reverting this unexport and adding a comment so no one else gets confused.